### PR TITLE
Adds a test which does a recursive copy.

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

File: src/server/pachyderm_test.go [hunk 1]
```diff
@@ -2033,6 +2033,48 @@ func TestDeleteAll(t *testing.T) {
 	require.Equal(t, 0, len(jobInfos))
 }
 
+func TestRecursiveCp(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	// this test cannot be run in parallel because it deletes everything
```
Comment (sjezewski) [inline src/server/pachyderm_test.go]: I dont think this comment doesn't apply anymore

**Comment by sjezewski** [comment]
Other than minor comment, LGTM

image

File: src/server/pachyderm_test.go (hunk 1 above)
Comment (jdoliner) [inline src/server/pachyderm_test.go]: Yup... classic copy pasta error.

**Comment by jdoliner** [comment]
meme lgtm

**Pull request merged**